NSS-Alleged-Donkeys/WorkforceManagement
# Mh4 employee detail

## Link to Ticket
closes # [Link](https://github.com/NSS-Alleged-Donkeys/WorkforceManagement/issues/4)

## Description of Proposed Changes
- Adds ability to view employee details
- Employee Details include the employee's full name, what computer manufacturer and make they are assigned, and what training programs they are involved in.

## Steps to Test

- ``` git fetch --all ```
- ``` git checkout MH4-EmployeeDetail  ```
- ``` run sql file in SQL server management studio to make sure the appropriate data is in the database. ```
- run all tests in test explorer

## Impacted Areas in Application
- Employee.cs
- EmployeeTests.cs
- EmployeeController.cs
- EmployeeDetailViewModel
- Details.cshtml
- Bangazon.sql


## Mentions @username
@MissPeperr 
@davidnicholastaylor 
@edwards-j 
@kimberly-bird 

## Definition of Done
- Fully commented
- All tests work
- All code peer reviewed
